api/redhatopenshift/HcpCluster.Management/hcpCluster-models.tsp

```diff
@@ -70,9 +70,13 @@ model HcpOpenShiftClusterProperties {
   @visibility(Lifecycle.Read, Lifecycle.Create, Lifecycle.Update)
   platform: PlatformProfile;
 
-  /** Configure cluter capabilities. */
+  /** Configure cluster capabilities. */
   @visibility(Lifecycle.Read, Lifecycle.Create)
   capabilities?: ClusterCapabilitiesProfile;
+
+  /** Configure ETCD. */
+  @visibility(Lifecycle.Read, Lifecycle.Create)
```

Comment:
Does this propagate to subsections? What happens if in the future we want to allow updating something inside the etcd section or subsections? will adding update be easy or will it be considered breaking somehow?